versioned_docs/version-8.2/self-managed/zeebe-deployment/security/client-authorization.md

```diff
@@ -7,11 +7,11 @@ description: "Learn how the Zeebe Gateway supports Camunda Identity-based auth t
 ## Gateway
 
 The Zeebe Gateway supports [Camunda Identity](../../identity/what-is-identity.md)-based auth token validation.
-By default, authentication is disabled by setting `security.authentication.mode` to `none`.
+In Camunda Platform 8 Self-Managed Helm chart, the authentication is enabled via Camunda Identity like all other applications.
 
 ### Camunda Identity authorization
 
-[Camunda Identity](../../identity/what-is-identity.md)-based OAuth token validation can be enabled by setting `security.authentication.mode` to `identity` and providing the corresponding `security.authentication.identity.*` properties. You can find more details about these in the [Gateway config documentation](../configuration/gateway.md#zeebegatewayclustersecurityauthenticationidentity).
+[Camunda Identity](../../identity/what-is-identity.md)-based OAuth token validation is already enabled by setting `security.authentication.mode` to `identity` and providing the corresponding `security.authentication.identity.*` properties. You can find more details about these in the [Gateway config documentation](../configuration/gateway.md#zeebegatewayclustersecurityauthenticationidentity).
```

Comment:
💭 Would leave this paragraph as is, just explaining how it's configured but add another one that hints about the default in the helm chart, wdyt?
```suggestion
[Camunda Identity](../../identity/what-is-identity.md)-based OAuth token validation can be enabled by setting `security.authentication.mode` to `identity` and providing the corresponding `security.authentication.identity.*` properties. You can find more details about these in the [Gateway config documentation](../configuration/gateway.md#zeebegatewayclustersecurityauthenticationidentity).

In the Camunda Platform 8 Self-Managed Helm chart it's already fully preconfigured by default.
```